lpc11u6x/README.md

```diff
@@ -0,0 +1,14 @@
+# lpc11u6x-rs
+
+Rust register mappings for the NXP LPC11U6x family of ARM Cortex-M0+
+microcontrollers. The code is generated automatically from the [SVD
+file](https://community.nxp.com/servlet/JiveServlet/downloadBody/334624-102-1-275615/LPC11U6x+SVD+file.zip)
+using [svd2rust](https://crates.io/crates/svd2rust) 0.15.
+
+LPC11U6x/LPC11E6x user manual: [http://www.mouser.com/ds/2/302/UM10732-315822.pdf](http://www.mouser.com/ds/2/302/UM10732-315822.pdf)
+
+This crate is part of the [lpc-pac](https://github.com/lpc-rs/lpc-pac/)
+project.
+
+This crate requires nightly Rust to compile since it uses `untagged_unions` to
+access registers which map to the same address.
```

Comment:
This is weird: Above, it says svd2rust 0.15 is used, here it says nightly is required (which you confirmed), but the [svd2rust changelog](https://github.com/rust-embedded/svd2rust/blob/master/CHANGELOG.md#v0150---2019-07-25) states that nightly is no longer required in 0.15, as unions are no longer generated.

One of these must be wrong.

Reply:
Well it is still generated using 0.14.0 here, but I already have another PR for upgrading svd2rust to the latest version, probably that's the reason for the inconsistency. Fixing the docs here, and I'll fix the conflict in the other PR when we get there.